### PR TITLE
Export Funct anno (all vars), cancer VAFs and COSMIC IDs (cancer vars) when downloading variants

### LINKED ISSUE #2398: Add VAF info exported variant excel file?
Hi! 
 When you export variants from the variant page to excel, the HGNC transcript does not come up, instead some other prefereed transcript comes up. Can this be changed, or can a column be added for the HGNC transcript? 

Hälsningar Hero

### LINKED ISSUE #3577: Include COSMIC ID and consequence in exported cancer variants files
COSMIC IDs and consequence are shown on the cancer variantS page, but not exported in files.

### PR DESCRIPTION
**Pull request by northwestwitch**
Fix #3577 and fix #2398

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the paxa software installed on `hasta`:
 - Log in into hasta: `ssh <USER.NAME>@hasta.scilifelab.se`
 - Activate the staging environment with the command `us` (Use Stage)
 - Run the command `paxa` and follow the instructions. Make sure to specify scout-stage as the resource you request and the server cg-vm1 as server.
1. Log out from the hasta server.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, log out from `cg-vm1` and log in again in the `hasta` server, repeat the `hasta` and `paxa` procedure, which will release the allocated resource (scout-stage) to be used for testing by other users.
</details>

**How to test**:
1. Go on scout-stage and try to download variants from RD and cancer cases
2. Make sure downloading works
3. Make sure a new "consequence" column is available in downloaded files
4. Make sure 3 new cols (VAF cancer, VAF normal, COSMIC ID) are now available in downloaded files from cancer cases

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DN
- [x] tests executed by DN

### CODE REVIEW

**Comment by northwestwitch** [comment]
This is basically ready, it would be nice to include it in the new release. What do you think @dnil ?

**Comment by dnil** [comment]
Sure, agreed - just ping me when you are ready for a review! I won't promise I have time today, but its perhaps also not super with a Friday release. 😊

**Comment by northwestwitch** [comment]
I got it fixed. This is ready for review now

File: tests/server/blueprints/variants/test_variants_controllers.py [hunk 1]
```diff
@@ -730,6 +732,47 @@ def test_variant_csv_export(real_variant_database, case_obj):
         assert len(export_cols) == len(export_header)
 
 
+def test_variant_csv_export_cancer(real_variant_database, case_obj):
```
Comment (dnil) [inline tests/server/blueprints/variants/test_variants_controllers.py:735]: 🥇

**Review by dnil — APPROVED** [review]
Looks good to me!

**Pull request merged**